CONTRIBUTORS.md

```diff
@@ -1,28 +1,46 @@
 # Contributors
```

Comment:
Is this section intended to be a running list of all contributors over time? I thought we were going to track this by release instead. Am I remembering that incorrectly?

Reply:
I think what we agreed to was that contributors is anyone who's contributed ever, whereas authors in manuscripts, posters, etc is defined as the people who contributed since the last release.